changes/31.5.1.md

```diff
@@ -0,0 +1 @@
+* Add Latin localization forms for Polish and Romanian.
```

Comment:
Change the version to 31.6.0. I want to better enforce SemVer -- given this is a new feature we should produce a minor version change.